cmd/storer2elastic/databasereader/persistersReader.go

```diff
@@ -0,0 +1,94 @@
+package databasereader
```

Comment:
because it uses all the inner fields of the databaseReader struct. just wanted to do a small separation by moving them to a new file. we have used this pattern also in other packages